refinery/data_set_manager/urls.py

```diff
@@ -59,4 +59,5 @@
         AddFileToNodeView.as_view()),
     url(r'^nodes/(?P<uuid>' + UUID_RE + r')/$',
         NodeViewSet.as_view()),
+    url(r'^studies/$', StudiesView.as_view()),
```

Comment:
Why not use the DefaultRouter like other instances?

Reply:
Routers require viewSets, and this api uses the APIView class. It's a limited API which only allows get lists, so I like to be explicit and use the APIView class.